src/main/scala/org/apache/sysml/api/ml/BaseSystemMLClassifier.scala

```diff
@@ -74,12 +74,14 @@ trait BaseSystemMLEstimatorOrModel {
   var statistics:Boolean = false
   var statisticsMaxHeavyHitters:Int = 10
   val config:HashMap[String, String] = new HashMap[String, String]()
+  val registeredInputs:HashMap[String, MatrixBlock] = new HashMap[String, MatrixBlock]()
```

Comment:
nitpick: Could we rename this to `inputs` since the idea of "registered inputs" is tied to the old MLContext?